files/fail_over.py

```diff
@@ -187,4 +224,7 @@ def items(self):
 
 
 if __name__ == "__main__":
-    FailOver().run('dr.conf', '/var/log/ovirt-dr/ovirt-dr.log')
+    level = logging.getLevelName("DEBUG")
+    conf = 'dr.conf'
+    log = '/var/log/ovirt-dr/ovirt-dr.log'
```

Comment:
This directory don't exist so it should be here.

Reply:
It's not, but anyone other testing it, will fail, because he won't have that directory.